terraform/spec/dependabot/terraform/file_fetcher_spec.rb

```diff
@@ -14,68 +14,30 @@
       directory: directory
```

Comment:
Terraform is set up to always clone the repo so I've updated the specs to use a clone when fetching files